tests/e2e_suite.py

```diff
@@ -139,6 +150,13 @@ def add(parser):
         parser.add_argument(
             "--test-duration", help="Duration of full suite (s)", type=int
         )
+        parser.add_argument(
```

Comment:
As we are extending the suite, I wonder if we should create a new class that records the different sub-suites. Something like:
```
s = Suites()
s.add_suite(suite_rekey_recovery, "description")
s.add_suite(suite_membership_recovery, "description")
```

`choices` here would be the list of all the suites that have been registered. And we could filter based on the selected `--test-suite`. Then we wouldn't have to add a new `elif` at the top of this file every time a new suite is added.

Reply:
We could do something like this, though we still need a name mapping (ie `s.add_suite("rekey_recovery", suite_rekey_recovery, ...)` that can be entered from the command line. I agree this should be ideally be in `test_suite.py` rather than `e2e_suite.py`, but I think it can just be a dict rather than a class?